src/nl/rubensten/texifyidea/window/PreviewForm.kt

```diff
@@ -0,0 +1,126 @@
+package nl.rubensten.texifyidea.window
+
+import com.google.common.io.Files
+import java.io.*
+import java.util.concurrent.TimeUnit
+import javax.imageio.ImageIO
+import javax.swing.*
+
+class PreviewForm {
+    var panel: JPanel? = null
+    private var equation_area: JTextArea? = null
+    private var output_area: JTextArea? = null
+    private var output_panel: JPanel? = null
+    private var equation_panel9: JPanel? = null
+    private var preview_panel: ImagePanel? = null
+
+    private val preamble = """
+    \usepackage{amsmath,amsthm,amssymb,amsfonts}
+    \usepackage{color}
+    """.trimIndent()
+
+    fun set_equation_text(equation_text: String) {
+        equation_area!!.text = equation_text
+
+        try {
+            val tmpdir = Files.createTempDir()
+            try {
+                val tmp_basename = tmpdir.path + "/equation"
+                val writer = PrintWriter(tmp_basename + ".tex", "UTF-8")
+
+                val tmp_content = """
+\documentclass{article}
+""" + preamble +
+                        """
+\pagestyle{empty}
+\begin{document}
+""" + equation_text +
+                        """
+\end{document}"""
```

Comment:
Indent to 1 level deeper than the val assignment (see below an example).

Also make sure to use string templates for better readability.

Rename to `tempContent`

```kotlin
                val tempContent = """
                    \documentClass{article}
                    $preamble
                    
                    \pagestyle{empty}
                    \begin{document}
                    $equation_text
                    
                    \end{document}
                """.trimIndent()
```